### PR TITLE
Pinned widgets work

### PR DESCRIPTION
**Pull request by t3chguy**
Fixes https://github.com/vector-im/element-web/issues/15424
Related https://github.com/vector-im/element-web/pull/15499

Adhoc build at https://riots.im/adhoc/widgets-demo2/

image
image

### CODE REVIEW

File: res/css/views/rooms/_AppsDrawer.scss [hunk 1]
```diff
@@ -47,53 +47,100 @@ $MiniAppTileHeight: 200px;
...
+            position: absolute;
+            left: 3px;
+            top: 50%;
+            transform: translate(0, -50%);
+
+            height: 64px; // to match width of the ones on roomlist
+            width: 4px;
+            border-radius: 4px;
+
+            content: ' ';
```
Comment (turt2live) [inline res/css/views/rooms/_AppsDrawer.scss:61]: we tend to just use `''` instead, but this works too.

File: src/components/views/rooms/AppsDrawer.js [hunk 2]
```diff
@@ -170,9 +244,12 @@ export default class AppsDrawer extends React.Component {
...
         const classes = classNames({
-            "mx_AppsDrawer": true,
-            "mx_AppsDrawer_fullWidth": apps.length < 2,
-            "mx_AppsDrawer_minimised": !this.props.showApps,
+            mx_AppsDrawer: true,
+            mx_AppsDrawer_fullWidth: apps.length < 2,
+            mx_AppsDrawer_resizing: this.state.resizing,
+            mx_AppsDrawer_has1: apps.length === 1,
+            mx_AppsDrawer_has2: apps.length === 2,
+            mx_AppsDrawer_has3: apps.length === 3,
```
Comment (turt2live) [inline src/components/views/rooms/AppsDrawer.js:252]: "has" implies that these would be `>=` checks. Maybe "only" instead?

File: src/resizer/item.ts [hunk 3]
```diff
@@ -1,5 +1,6 @@
 /*
 Copyright 2019 New Vector Ltd
+Copyright 2020 The Matrix.org Foundation C.I.C.
```
Comment (turt2live) [inline src/resizer/item.ts:3]: merge NV copyright into the foundation

File: src/resizer/resizer.ts [hunk 4]
```diff
@@ -1,6 +1,6 @@
 /*
 Copyright 2018 New Vector Ltd
-Copyright 2019 The Matrix.org Foundation C.I.C.
+Copyright 2019, 2020 The Matrix.org Foundation C.I.C.
```
Comment (turt2live) [inline src/resizer/resizer.ts:3]: merge copyrights

**Review by turt2live — APPROVED** [review]

**Review by nadonomy — APPROVED** [review]
LGTM

**Pull request merged**